src/blueapi/service/main.py

```diff
@@ -43,7 +43,11 @@ def get_plans(handler: Handler = Depends(get_handler)):
     )
 
 
-@app.get("/plans/{name}", response_model=PlanModel)
+@app.get(
+    "/plans/{name}",
+    response_model=PlanModel,
+    responses={status.HTTP_404_NOT_FOUND: {"item": "not found"}},
```

Comment:
I think the default if the method throws a Key of Value error is "detail: "item not found", if we want to have it the same for neatness sake?